switchbot_ros/package.xml

```diff
@@ -28,5 +27,6 @@
   <build_export_depend>actionlib_msgs</build_export_depend>
 
   <export>
+    requirements.txt
```

Comment:
We have to have `<pip_requrements>` tag here.
https://github.com/locusrobotics/catkin_virtualenv/blob/4af99703e6c7faf12c5d328dbab346250fd085f7/catkin_virtualenv/src/catkin_virtualenv/collect_requirements.py#L29C43-L38

```suggestion
    <pip_requirements>requirements.txt</pip_requirements>
```